pyleague/simulation.py

```diff
@@ -185,3 +185,17 @@ def generate_result(team_a: models.Team, team_b: models.Team, outcome: str) -> T
     else:
         goals_home, goals_away = get_drawing_score(team_a, team_b)
     return goals_home, goals_away
+
+def generate_time_for_goals(goals_h: int, goals_a: int) -> List[List[int], List[int]]:
+    shuffle_space = [i for i in range(1, 91)]
+    mins_goal_a = fisher_shuffle(goals_a, shuffle_space, i = 0)
+    mins_goal_h = fisher_shuffle(goals_h, shuffle_space, i = goals_a)
```

Comment:
Your implementation is making me think of a new solution very similar to this that I didn't think about: we could take the `mins_goal_h` with `random.choices` with a correct parameter `k`. Then we could do the same with the `mins_goal_a`. Actually you can decide to take the minutes of all the goals (home and away) with a single call or to make two calls after managing the fact that we need to remove the mins of home team from the list before taking the minutes of the away team (probably the first solution would result in cleaner code).

Reply:
understood. Will check this out